libr/core/vasm.c

```diff
@@ -40,7 +42,7 @@ static int readline_callback(void *_a, const char *str) {
 		if (a->acode) {
 			xlen = strlen (a->acode->buf_hex);
 			strcpy (a->codebuf, a->blockbuf);
-			memcpy (a->codebuf, a->acode->buf_hex, xlen);
+			memcpy (a->codebuf, a->acode->buf_hex, R_MIN (xlen, R_VISUAL_ASM_BUFSIZE - 1));
```

Comment:
This will not be null terminated. Asan may complain

Reply:
Oops, you're totally right ^^ Should be fixed now